testing/submit_test.sh

```diff
@@ -21,10 +20,14 @@ echo
 
 # Need to do this so that the catalogue parameter file reflects the absolute path.
 # Otherwise the HBT reader will not work.
-sed -i "s|hbt_output|$PWD\/test_output|g" config_test.txt
+sed -i "/.*SubhaloPath.*/c\SubhaloPath ${OUTPUTDIR}" config_test.txt
 
 export OMP_NUM_THREADS=$SLURM_CPUS_PER_TASK
-mpirun -np $SLURM_NTASKS ./build/HBT config_test.txt
+mpirun $RP_OPENMPI_ARGS -np $SLURM_NTASKS ./build/HBT config_test.txt
```

Comment:
Perhaps we should add a condition to check whether to trigger the code below. Sometimes the code crashed and still said "Done". Made it a bit confusing.

Reply:
Putting a "set -e" in near the top will make the whole script abort if a command fails.

Reply:
Added `set - e` after after the modules have been loaded.